crates/turbopack-ecmascript-runtime/src/browser_runtime.rs

```diff
@@ -35,12 +35,12 @@ pub async fn get_browser_runtime_code(
     let runtime_backend_code = embed_static_code(
         asset_context,
         match chunk_loading {
-            ChunkLoading::Edge => "dev/runtime/edge/runtime-backend-edge.ts".into(),
+            ChunkLoading::Edge => "browser/runtime/edge/runtime-backend-edge.ts".into(),
```

Comment:
How can it be browser and edge at the same time?

Reply:
That's a refactor I'm still working on, previously `get_browser_runtime_code()` was called `get_dev_runtime_code` but it never used this for Node.js, instead it used `get_build_runtime_code`. Bit confusing naming so I've been working with @sokra to get this code into the right place.